rwengine/src/objects/GameObject.cpp

```diff
@@ -23,12 +23,11 @@ void GameObject::setRotation(const glm::quat& orientation) {
 }
 
 float GameObject::getHeading() const {
-    auto hdg = glm::roll(getRotation());
-    return hdg / glm::pi<float>() * 180.f;
+    auto headingInRadians = glm::roll(getRotation());
+    return glm::degrees(headingInRadians);
```

Comment:
nit: the idea was also to keep `setHeading` and `getHeading` in sync. If both use the same variable name for the same content, it's easier to guess what goes on. anyhow, I'm fine with the current solution, maybe just something to keep in mind for next time :)